packages/trace-viewer/src/ui/workbench.tsx

```diff
@@ -316,59 +320,67 @@ export const Workbench: React.FunctionComponent<{
     component: <MetadataView model={model}/>
   };
 
-  return <div className='vbox workbench' {...(inert ? { inert: 'true' } : {})}>
-    {!hideTimeline && <Timeline
-      model={model}
-      consoleEntries={consoleModel.entries}
-      boundaries={boundaries}
-      highlightedAction={highlightedAction}
-      highlightedEntry={highlightedEntry}
-      highlightedConsoleEntry={highlightedConsoleMessage}
-      onSelected={onActionSelected}
-      sdkLanguage={sdkLanguage}
-      selectedTime={selectedTime}
-      setSelectedTime={setSelectedTime}
-    />}
-    <SplitView
-      sidebarSize={250}
-      orientation={sidebarLocation === 'bottom' ? 'vertical' : 'horizontal'} settingName='propertiesSidebar'
-      main={<SplitView
-        sidebarSize={250}
-        orientation='horizontal'
-        sidebarIsFirst
-        settingName='actionListSidebar'
-        main={<SnapshotTabsView
-          action={activeAction}
+  return <SplitView
```

Comment:
This is now pretty difficult to parse given the number of stacked `SplitView`s. Can we break this up into separate components with clear naming?

Reply:
You're right, it's getting unwieldy. Separate components would be a bigger refactoring, but I extracted `workbench` into a named variable. Does that work for you?

Reply:
Yes. Much better